content/en/monitors/guide/how-to-set-up-rbac-for-monitors.md

```diff
@@ -0,0 +1,216 @@
+
+You might however have a number of existing monitors still leveraging the locked mechanism under the hood. This mechanism is still supported and respected for now, which means locked monitors should not be editable by users who do not have the [Datadog Admin Role][2] attached or are not the monitor's creator.
+
+Below sections describe how the change from the old locked mechanism to the new role restriction one is performed, depending on the way you manage your monitors.
+
+### API
+
+Although deprecated, the `locked` parameter corresponding to the above mentioned locked mechanism is still supported. This means you can progressively update the definition of your monitors managed through API/Terraform to stop using `locked` and start using `restricted_roles` (parameter attached with the new role restriction option).
+
+For more information on how to update your monitors' definitions, see the [Edit a monitor API endpoint documentation][3] and the dedicated [Monitor API Options guide][4].
+
+### UI
+
+All new monitors created from the UI use the new `restricted_roles` parameter. 
+All monitors (existing and new) also display the new role restriction option regardless of the mechanism used under the hood:
+
+{{< img src="/monitors/guide/monitor_rbac_non_restricted.jpg" alt="RBAC Non Restricted Monitor"  >}}
+
+Existing monitors have their definitions updated from the old locked mechanism to the new role restriction one on the fly whenever a monitor is saved.
+
+Below are some instructions to help you determine how to proceed in case you need to save a monitor that is using the previous locked mechanism.
+
+#### Locked monitors (`locked:true`) edited by creator or user with Datadog Admin Role
+
+You are a user with [Datadog Admin Role][2] and/or are the creator of the monitor. You are editing a locked monitor and seeing the below warning:
+
+<TO DO: ADD SCREENSHOT OF WARNING>
+
+On save, your monitor's definition will be updated to all roles with Admin permissions.
+You have different ways to handle this warning depending on the changes you are willing to make on your monitor:
+
+**1. You do not want to change anything about your monitor permissions**
+
+Save the monitor to have the update from the locked mechanism to the restricted_roles one be performed automatically, along with any other updates you might have done on your monitor (e.g. threshold update, message, etc.).
+
+If you were just opening the edit page to see more details about your monitor configuration, you can also hit **Save**. That also performs the update for your monitor.
+
+**2. You want to allow all users to edit this monitor**
+
+Save the monitor, edit it again, and remove all roles that were generated for this monitor. 
+
+**3. You want your monitor to be restricted to some roles, but not the exact equivalent of the previous locked option**
```

Comment:
```suggestion
**3. You want to change the roles allowed to modify your monitor**
```

Reply:
This makes it sound like the monitor is already using the restricted role mechanism at this point, which it's not.